src/box-container/box-container.tsx

```diff
@@ -107,27 +120,40 @@ export const BoxContainer = ({
                 <Handle
                     onClick={onHandleClick}
                     type="button"
-                    aria-label={showExpanded ? "Collapse" : "Expand"}
+                    aria-label={
+                        showExpanded
+                            ? `${title} expanded`
+                            : `${title} collapsed`
```

Comment:
we can use `aria-labelledby` here? and we could remove the `expanded/collapsed` labels since the state is announced by `aria-expanded`